system/Test/CIDatabaseTestCase.php

```diff
@@ -135,6 +142,8 @@ public function __construct()
 	 */
 	public function setUp()
 	{
+		$this->loadDependencies();
```

Comment:
I made the method, because when the logic was in the constructor, it was executed (probably PHPUnit creates the test case instance) and caused database connection errors (if you don't have correct database config or don't have running database).
So to avoid the errors, I moved it to `setUp()`.

First I tried to move it to `setUpBeforeClass()`, but it was static method.